### PR TITLE
[Go] Update gin to 1.4.0

### PR DESCRIPTION
**Pull request by appleboy**
As title. closed https://github.com/the-benchmarker/web-frameworks/pull/1277

Signed-off-by: Bo-Yi Wu <appleboy.tw@gmail.com>

### CODE REVIEW

File: go/gin/go.mod [hunk 1]
```diff
@@ -1,11 +1,5 @@
...
-require (
-	github.com/gin-contrib/sse v0.0.0-20190301062529-5545eab6dad3 // indirect
-	github.com/gin-gonic/gin v1.3.0
-	github.com/golang/protobuf v1.3.0 // indirect
-	github.com/mattn/go-isatty v0.0.6 // indirect
-	github.com/ugorji/go/codec v0.0.0-20190204201341-e444a5086c43 // indirect
-	gopkg.in/go-playground/validator.v8 v8.18.2 // indirect
-	gopkg.in/yaml.v2 v2.2.2 // indirect
-)
+go 1.12
```
Comment (waghanza) [inline go/gin/go.mod]: You can explicitly define the `go` **version** ?

File: go/gin/go.mod [hunk 2]
```diff
@@ -1,11 +1,5 @@
...
+
+require github.com/gin-gonic/gin v1.4.0
```
Comment (waghanza) [inline go/gin/go.mod]: :tada:

File: go/gin/go.sum [hunk 3]
```diff
@@ -0,0 +1,38 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
```
Comment (waghanza) [inline go/gin/go.sum]: How do you get this file ? By a `go get` ?

File: go/gin/go.mod (hunk 1 above)
Comment (appleboy) [reply go/gin/go.mod]: Yes. you can try the following command for initiating the project.

```
$ go mod init xxxx
```

File: go/gin/go.sum (hunk 3 above)
Comment (appleboy) [reply go/gin/go.sum]: Just try the `go mod tidy`

**Pull request merged**